### ISSUE TITLE
Open SQLite database file with drag&drop

### ISSUE THREAD

**Issue by nmoreaud** [dbeaver/dbeaver#18322]
I would like to be able to open SQLite files easily by drag&drop from windows explorer to DBeaver.
Currently it opens a query tab with binary content.

**Comment by ShadelessFox** [comment]
Hello @nmoreaud,

Thank you for your request.

We can add a DND handler in the navigator that will prompt the user to create a new connection.

**Comment by 2-5** [comment]
I would also like this. And also for DuckDB files.

Instead of prompting the user I think it would be better to just create a new connection with default (or last used) settings.

**Comment by nmoreaud** [comment]
Yeah, automatically create a connection with default parameters would be fine

**Comment by meteor1113** [comment]
I would also like this feature.
like datagrip: https://www.jetbrains.com/guide/databases/tips/create-sqlite-connection/?playlist=A
